applications/DEMApplication/python_scripts/sphere_strategy.py

```diff
@@ -324,11 +324,10 @@ def SolveSolutionStep(self):
         self.FixExternalForcesManually(time)
         (self.cplusplus_strategy).Solve()
 
-    def AdvanceInTime(self, step, time, is_time_to_print = False):
-        step += 1
+    def AdvanceInTime(self, time):
```

Comment:
API change. warning for deprecated method, please.

Reply:
+1 for @maceligueta 

Since this procedure is widely used in Kratos in the meantime I might add a global function for doing this (note to myself)